src/components/ContestTile.js

```diff
@@ -19,9 +19,8 @@ const ContestTile = ({ contest: { node }, updateContestStatus }) => {
     findingsRepo,
     repo: contestRepo,
     fields,
-  } = node;
-  const { submissionPath, contestPath } = fields;
-
+    status,
+  } = contest;
```

Comment:
If going with the modifications.

```suggestion
  } = contest.node;
```

Reply:
Same comment as above :)